packages/website/src/pages/playground.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Layout from "@theme/Layout";
+import { SquigglePlayground } from "../components/SquigglePlayground";
+
+export default function PlaygroundPage() {
+  return (
+    <Layout title="Playground" description="Squiggle Playground">
+      <div
+        style={{
+          maxWidth: 2000,
```

Comment:
I made this pretty big, so that those with large monitors could use them for it. However, the component right now isn't optimized for larger displays. (More changes will come)